codegen/projections/white_label/lib/white_label/builders.rb

```diff
@@ -10,6 +10,20 @@
 module WhiteLabel
   module Builders
 
+    # Operation Builder for DefaultKitchenSink
```

Comment:
These should be alphabetical too. Not sure if that broke during this PR or we had a weird ordering to begin with.

Reply:
fixed